src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -258,7 +258,7 @@ protected static bool IsFieldOrPropertyReference(IOperation operation, INamedTyp
         protected class MyCodeAction : CodeAction.DocumentChangeAction
         {
             public MyCodeAction(string title, Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(title, createChangedDocument)
+                : base(title, createChangedDocument, equivalenceKey: title)
```

Comment:
Specifying an equivalence key can't cause any other issues, right? I don't really know what they're for :)

Reply:
Specifying the equivalence key is good, but it's preferable to use a non-localized name for it. Personally I like using `nameof(Resources.Whatever_the_title_is)`.